opensearch-operator/pkg/builders/cluster_test.go

```diff
@@ -207,6 +208,22 @@ var _ = Describe("Builders", func() {
 		})
 	})
 
+	When("When Reconciling the cluster reconciler", func() {
```

Comment:
The description is misleading as you are not running the reconciler, you are just running the specific builder

Reply:
Updated the description to this test and other added unit tests.
Thank you